addons/l10n_pk_edi/models/product_product.py

```diff
@@ -0,0 +1,204 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from odoo import _, models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    l10n_pk_edi_sale_type = fields.Selection(
+        selection=[
```

Comment:
I would keep selection data and model definition separated, it's more readable.

```suggestion

SALE_TYPES = [
    # [...]
]

class ProductTemplate(models.Model):
    _inherit = 'product.template'

    l10n_pk_edi_sale_type = fields.Selection(selection=SALE_TYPES, string="Sale Type (FBR)", required=True, default='T1000024')
```